package.json

```diff
@@ -16,9 +16,9 @@
     "glob": "^7.1.3",
     "hash-index": "^3.0.0",
     "inquirer": "^6.3.1",
+    "react-native-rename": "github:panwrona/react-native-rename",
```

Comment:
Looks like a temporary change for development. Do these changes work with unmodified react-native-rename?